tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "target": "ES2017",                      
+    "module": "CommonJS",                    
+    "declaration": true,                  
+    "sourceMap": true,                  
+    "strict": true,                          
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "esModuleInterop": true,
+    "outDir": "dist",
+    "typeRoots": [
+      "./node_modules/@types",
+      "src/dependencies.d.ts"
```

Comment:
There's nothing wrong with this. A potential improvement in the future would be to create a 'types' directory in the root of the project, and point your additional `typeRoots` to that directory. Then it will mirror the behavior of `node_modules`. `types/ethjs/index.d.ts` would be loaded for types for `ethjs`. See https://github.com/MetaMask/browser-passworder/tree/master/types for an example.

Reply:
We don't have an established convention for this, but I think we should, so let's do this!